webextensions/manifest/author.json

```diff
@@ -6,7 +6,7 @@
           "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/author",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "≤58"
             },
             "edge": {
```

Comment:
The linter has not stated that the data should be mirrored, which means the Edge data we have is different than what's generated by the mirroring script.  It's probably because of the ranges, if I had to guess!